src/Http/Controllers/VoyagerBaseController.php

```diff
@@ -996,4 +988,17 @@ protected function relationIsUsingAccessorAsLabel($details)
     {
         return in_array($details->label, app($details->model)->additional_attributes ?? []);
     }
+
+    protected function applyScopes( $dataType, $model, $query )
+    {
+        if (isset($dataType->scope) && !empty($dataType->scope)) {
+            if( ! is_array($dataType->scope))
+                $dataType->scope = [$dataType->scope];
+            foreach ($dataType->scope as $scope)
+            {
+                if (method_exists($model, 'scope'.ucfirst($scope)))
+                    $query->{$scope}();    
+            }
+        }
+    }
```

Comment:
I find it could be more readable this way:

```php
protected function applyScopes($dataType, $model, $query)
{
    if (empty($dataType->scope)) {
        return;
    }

    foreach (Arr:wrap($dataType->scope) as $scope) {
        if (method_exists($model, 'scope'.ucfirst($scope))) {
            $query->{$scope}();
        }
    }
}
```

No need to use `isset` if you are using `empy` and less levels of nesting.